sections/product.liquid

```diff
@@ -102,7 +102,7 @@
           "label": "Title",
           "id": "stock_title",
           "type": "text",
-          "default": "المنتجات المتبقية في المخزون"
+          "default": "Items left in stock"
```

Comment:
Since in other pages we have defaults in Arabic, shouldn't this remain in Arabic?

Reply:
no check the ticket description. we already include that we should add  `Items left in stock ` as default text

Reply:
Yes but the that's for the property title, i.e what the seller will see in the theme editor, since the store is in Arabic the default value should be in Arabic